test/unit-test/mqtt_agent_utest.c

```diff
@@ -93,6 +93,11 @@ static uint8_t packetType;
  */
 static uint16_t packetIdentifier;
 
+/**
+ * @brief Mock Counter variable for calling stubReceive multiple times.
```

Comment:
Forgot to update this
```suggestion
 * @brief Mock Counter variable for calling stubReceiveThenFail multiple times.
```